bin/environmentd-debug-zip

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# materialized — build and run a core service or test.
+
+if [ -z "${MZDEV_POSTGRES}" ]; then
+    POSTGRES='postgres://root@localhost:26257/materialize'
+else
+    POSTGRES=${MZDEV_POSTGRES}
+fi
+
+echo "Shutting down running environmentd and clusterd (if any)"
+killall -q environmentd
+killall -q clusterd
```

Comment:
I believe `killall` takes multiple procnames in the same invocation.

```suggestion
killall -q environmentd clusterd
```

Reply:
Confirmed, done!